mvlearn/decomposition/groupica.py

```diff
@@ -235,19 +242,29 @@ def transform(self, Xs, y=None):
         """
         check_is_fitted(self)
         Xs = check_Xs(Xs, copy=True)
+        if index is None:
+            index_ = np.arange(len(self.means_))
```

Comment:
```suggestion
            index_ = np.arange(self.n_views_)
```
Might as well use `n_views_` if we store it